### ISSUE TITLE
Issue in JDK8 for notarization Apple

### ISSUE THREAD

**Issue by StefanBas** [AdoptOpenJDK/openjdk-support#149]
OpenJDK8 just works well but there are some artifacts left.
As not needed for me, i deleted them. Can you please have a look at it?
 
 "issues": [
    {
      "severity": "error",
      "code": null,
      "path": "...MacOSX.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/hsdb",
      "message": "The binary uses an SDK older than the 10.9 SDK.",
      "docUrl": null,
      "architecture": "x86_64" 
    },
    {
      "severity": "error",
      "code": null,
      "path": "....MacOSX.app/Contents/PlugIns/JRE/Contents/Home/jre/bin/clhsdb",
      "message": "The binary uses an SDK older than the 10.9 SDK.",
      "docUrl": null,
      "architecture": "x86_64" 
    }
  ]
}

**Comment by aahlenst** [comment]
@gdams Could you please have a look?

**Comment by aahlenst** [comment]
I'm a bit suprised to see `The binary uses an SDK older than the 10.9 SDK.` in there because we are building with 10.14 SDK. What version of AdoptOpenJDK 8 did you use and what steps did you perform?

**Comment by StefanBas** [comment]
I am using Mojave and i am builing a project with JavaFX and AdoptOpenJDK. First i used the last version of OpenJDK built with SDK < 10.9, i think it was something like jdk8u242-b08. I ugraded this version with the latest pkg installer to jdk8u265-b01. And built again. These 2 failure where left when notarizing the app.
Maybe updating an older version of AdoptOpenJKDK led to this issue.

**Comment by aahlenst** [comment]
JDK PKG (upgraded over time from 232 or maybe even earlier):

```
$ /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/bin/java -version
openjdk version "1.8.0_265"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_265-b01)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.265-b01, mixed mode)
$ find /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk -name "hsdb"  
/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/bin/hsdb
$ otool -l /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/bin/hsdb | grep -A 3 LC_VERSION_MIN_MACOSX
      cmd LC_VERSION_MIN_MACOSX
  cmdsize 16
  version 10.9
      sdk 10.14
$ find /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk -name "clhsdb"
/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/bin/clhsdb
$ otool -l /Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/bin/clhsdb | grep -A 3 LC_VERSION_MIN_MACOSX
      cmd LC_VERSION_MIN_MACOSX
  cmdsize 16
  version 10.9
      sdk 10.14
```

Both files aren't in the JRE at all:

```
$ find jdk8u265-b01-jre -name "hsdb"
$ find jdk8u265-b01-jre -name "clhsdb"
```

So it looks like something is happening between installation and notarization.

**Comment by StefanBas** [comment]
See imgae of cleaninstall of latest release. There are these two files.
![Bildschirmfoto 2020-08-03 um 09 53 32](https://user-images.githubusercontent.com/68897087/89159123-5d340d80-d56f-11ea-810c-62b843aa582f.png)

**Comment by StefanBas** [comment]
Ok, maybe my fault, do i have to refer to ../Contents/Home/jre for bundling the jre? 
But nevertheless, these two files lead to the SDK <10.9 error when notarizing.

**Comment by gdams** [comment]
hmm this is strange, those binaries should all be built by the same toolchain level...

**Comment by StefanBas** [comment]
One thing i forgot, before notarization i perform following for signing:
... itdev$ codesign --verbose --force --sign "Developer ID Application: (xxxxxxxxxx)" --timestamp --deep --options=runtime MacOSX.app/Contents/MacOS/JavaAppLauncher

**Comment by aahlenst** [comment]
@StefanBas Use otool as I did above to get the SDK those files have been compiled with. Also, unpack the pkg you've produced, use otool on the files in the pkg and also run `java -version` on the files extracted from the pkg you have created. I suspect either:

* Leftovers from a previous build on your machine are included that should have been updated
* Accidental inclusion of an old JDK from somewhere else on your machine during your build

**Comment by StefanBas** [comment]
Ok, i think there was something strange going on with my build process.
I rebuilt my app with clean install of AdoptOpenJDK and signed it again for notarization.
I looked at the binaries in the JRE and both binaries mentioned above where gone.
Good for you and strange for me :-)
Maybe something like

-     Leftovers from a previous build on your machine are included that should have been updated
-     Accidental inclusion of an old JDK from somewhere else on your machine during your build (Yes, could have been ZULU)

Sorry, but i think the Ticket can be closed as it was something strange on my machine. 
Best regards Stefan
